xml/art_virtualization-best-practices.xml

```diff
@@ -3241,6 +3241,162 @@ LC_ALL=C PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin \
       </para>
 <screen>&prompt.root;<command>echo 1 > /sys/devices/system/cpu/cpu<replaceable>X</replaceable>/online</command></screen>
     </sect2>
+
+    <sect2 xml:id="sec-vm-guest-scsi-pr">
+      <title>SCSI3 persistent reservation (S3-PR) on a multipathed device</title>
+      <para>
+        SCSI persistent reservations allow restricting access to block devices
+        to specific initiators in a shared storage setup.  When implementing
+        clustering of &vmguest;, it is a common requirement for &vmguest; to
+        send persistent reservation SCSI commands.  However, the operating
+        system restricts sending these commands to unprivileged programs
+        because incorrect usage can disrupt regular operation of the storage
+        fabric. QEMU's SCSI passthrough devices <literal>scsi-block</literal>
+        and <literal>scsi-generic</literal> support passing guest persistent
+        reservation requests to a privileged external helper program.
+        The <literal>qemu-pr-helper</literal> is that external helper, It needs
+        to start before QEMU, creates a listener socket which will accept
+        incoming connections for communication with QEMU.
+      </para>
+      <para>
+        Proper support of persistent reservation for multipath devices requires
```

Comment:
Again, Commercial users don't need to care about qemu level, just need to correctly setup the disk element in vm config, then libvirt handles everything.
I describes these just because hope presenting a brief introduction about the qemu scsi pr help program.